src/test/java/org/example/LogisticAppTest.java

```diff
@@ -0,0 +1,38 @@
+package org.example;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class LogisticAppTest
+    extends TestCase
```

Comment:
add tests for the important parts like checking the sorting and the calculations on different combinations of packages